qsimcirq_tests/qsimcirq_test.py

```diff
@@ -83,6 +84,38 @@ def test_cirq_qsim_simulate_fullstate(self):
     assert cirq.linalg.allclose_up_to_global_phase(
         result.state_vector(), cirq_result.state_vector())
 
+  def test_cirq_qsim_simulate_random_unitary(self):
+
+    q0, q1 = cirq.LineQubit.range(2)
+    qsimSim = qsimcirq.QSimSimulator(qsim_options={'t': 16, 'v': 0})
+    for _ in range(10):
+        random_circuit = cirq.testing.random_circuit(qubits=[q0, q1],
+                                                     n_moments=8,
+                                                     op_density=0.99)
+        cirq.ConvertToCzAndSingleGates().optimize_circuit(random_circuit) # cannot work with params
+        cirq.ExpandComposite().optimize_circuit(random_circuit)
+        qsim_circuit = qsimcirq.QSimCircuit(random_circuit)
+        circuit_unitary = []
+        for x in range(4):
```

Comment:
What is the intent of this loop? AFAIK qsim is deterministic, so every iteration will produce the same result.

Reply:
Good point. This loop was originally meant to test the 2-qubit circuit with each of the four initial states. But qsim does not support initial states yet so this loop is unnecessary.